Mathlib/Data/Set/Pointwise/SMul.lean

```diff
@@ -1034,6 +1034,22 @@ lemma inv_op_smul_set_distrib (a : α) (s : Set α) : (op a • s)⁻¹ = a⁻¹
 lemma smul_set_disjoint_iff : Disjoint (a • s) (a • t) ↔ Disjoint s t := by
   simp [disjoint_iff, ← smul_set_inter]
 
+example (a : α) (s : Set β) : a • s = (fun b ↦ a • b) '' s := by
+  rw [@image_smul]
+
+@[to_additive]
+lemma smul_set_iInter {ι : Type*} (a : α) (s : ι → Set β) :
+    (a • ⋂ i, s i) = ⋂ i, a • s i := by
```

Comment:
This is a special case of [Set.image_iInter](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Data/Set/Lattice.html#Set.image_iInter)